src/megaapi_impl.cpp

```diff
@@ -10394,6 +10418,16 @@ void MegaApiImpl::login_result(error result)
     MegaRequestPrivate* request = requestMap.at(client->restag);
     if(!request || (request->getType() != MegaRequest::TYPE_LOGIN)) return;
 
+    // if login with user+pwd succeed, update lastLogin timestamp
+    if (result == API_OK && request->getEmail() &&
+            (request->getPassword() || request->getPrivateKey()))
+    {
+        int creqtag = client->reqtag;
+        client->reqtag = 0;
+        updatePwdReminderData(false, false, false, false, true);
+        client->reqtag = creqtag;
```

Comment:
I think that this isn't going to work as expected. `updatePwdReminderData` adds a new request to the queue, and those requests take the tag when they are extracted from the request queue by the SDK thread, so I think that the request would take a valid tag anyway, despite the code above seems to try to avoid it. It would be needed to directly use `MegaClient::putua`.

Reply:
Yes, you're right. My intention was to perform the request internally, with the tag = 0, but it's true that it's not working like that.
I'll simply allow the new request to get its tag as usual, unless you consider that `tag=0` is worthy.